pysat/instruments/omni_hro.py

```diff
@@ -222,8 +222,8 @@ def calculate_imf_steadiness(inst, steady_window=15, min_window_frac=0.75,
         inst['clock_angle'].rolling(min_periods=min_wnum,
                                     window=steady_window,
                                     center=True).apply(pystats.nan_circstd,
-                                                       kwargs=circ_kwargs,
-                                                       raw=True)
+                                                       kwargs=circ_kwargs)
+                                                       #raw=True)
```

Comment:
Added `raw=True` back in #173.  Now required as part of pandas.  The True option sends the ndarray directly to the `rolling.apply` function.

From https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.core.window.Rolling.apply.html
```
The raw parameter is required and will show a FutureWarning if not passed. In the future raw will default to False.

New in version 0.23.0.
```

Reply:
fixed